src/.vuepress/config.js

```diff
@@ -51,7 +51,7 @@ const sidebar = {
       ]
     },
     {
-      title: 'Reusability & Composition',
+      title: 'Reusabilidad y Composición',
```

Comment:
Again, just to keep it consistent 😅 
```suggestion
      title: 'Reusabilidad & Composición',
```